src/network/announcethread.py

```diff
@@ -25,9 +30,14 @@ def run(self):
                 self.stop.wait(10)
 
     def announceSelf(self):
+        """Announce our presence"""
+        # pylint: disable=no-self-use
```

Comment:
Maybe `@staticmethod` was the right decision, why did you change?

Reply:
Might be, it was mistakenly done.